bin/database.py

```diff
@@ -635,6 +635,21 @@ def dm_v2_updates(cont_list, cont_name):
     query['$or'].append({'measurement': { '$exists': True}})
     dm_v2_updates(config.db.acquisitions.find(query), 'acquisitions')
 
+
+def upgrade_to_22():
+    """
+    scitran/core PR #397
+
+    Add access logging
+    """
+    config.db.singletons.find_one_and_update(
+        {'_id': 'config', 'persistent.db_log_uri': {'$exists': False}},
+        {'$set': {
+            'persistent.db_log_uri': config.get_item('persistent', 'db_log_uri'),
+            'core.access_log_enabled': config.get_item('core', 'access_log_enabled')
+        }})
```

Comment:
Dumb question - this should have the affect of inserting default values in the database, correct?

Does @ryansanford prefer that method? Will you have pre-populated values before the upgrade runs?